e2e_playwright/label_markdown_test.py

```diff
@@ -18,7 +18,7 @@
 
 
 def test_button_labels_handle_markdown(
-    app: Page, assert_snapshot: ImageCompareFunction
+    app: Page, themed_app: Page, assert_snapshot: ImageCompareFunction
```

Comment:
There should be only one app fixture per test function. Otherwise, this can explode the number of test executions per function. In this case it's better to just have a single test function that runs against themed app and have all other functions just run with `app`.